aria-practices.html

```diff
@@ -812,81 +814,90 @@ <h5>Textbox Keyboard Interaction</h5>
+            For example, in a messaging application, the default action may be to add the accepted value to a list of message recipients and then clear the combobox so the user can add another recipient.
+          </li>
+          <li>Printable Characters:
+            <ul>
+              <li>If the combobox is editable, type characters in the combobox. Note that some implementations may regard certain characters as invalid and prevent their input.</li>
+              <li>If the combobox is not editable, optionally moves focus to a value that starts with the typed characters.</li>
+            </ul>
           </li>
-          <li>Printable Characters: Type characters in the textbox. Note that some implementations may regard certain characters as invalid and prevent their input.</li>
-          <li>Standard single line text editing keys appropriate for the device platform (see note below).</li>
+          <li>If the combobox is editable, it supports standard single line text editing keys appropriate for the device platform (see note below).</li>
           <li><kbd>Alt + Down Arrow</kbd> (Optional): If the popup is available but not displayed, displays the popup without moving focus.</li>
           <li><kbd>Alt + Up Arrow</kbd> (Optional): If the popup is displayed:
-            <ol>
-              <li>If the popup contains focus, returns focus to the textbox.</li>
+            <ul>
+              <li>If the popup contains focus, returns focus to the combobox.</li>
               <li>Closes the popup.</li>
-            </ol>
+            </ul>
           </li>
         </ul>
-        <p class="note">Standard single line text editing keys appropriate for the device platform:</p>
-        <ol>
-          <li>include keys for input, cursor movement, selection, and text manipulation.</li>
-          <li>Standard key assignments for editing functions depend on the device operating system.</li>
-          <li>The most robust approach for providing text editing functions is to rely on browsers, which  supply them for HTML inputs with type text and for elements with the <code>contenteditable</code> HTML attribute.</li>
-          <li><strong>IMPORTANT:</strong> Be sure that JavaScript does not interfere with browser-provided text editing functions by capturing key events for the keys used to perform them.</li>
-        </ol>
+        <div class="note">
+          <p>Standard single line text editing keys appropriate for the device platform:</p>
+          <ol>
+            <li>include keys for input, cursor movement, selection, and text manipulation.</li>
+            <li>Standard key assignments for editing functions depend on the device operating system.</li>
+            <li>The most robust approach for providing text editing functions is to rely on browsers, which  supply them for HTML inputs with type text and for elements with the <code>contenteditable</code> HTML attribute.</li>
```

Comment:
```suggestion
            <li>The most robust approach for providing text editing functions is to rely on browsers, which supply them for HTML inputs with type text and for elements with the <code>contenteditable</code> HTML attribute.</li>
```
Remove space between "which supply."